### ISSUE TITLE
Support providing an alternative URL base for media paths

### ISSUE THREAD

**Issue by towerofnix** [hsmusic/hsmusic-wiki#121]
Individual builds of the site should not require uploading (accessing) the entire media repository in a sibling directory to other site files (i.e, as a subdirectory under the site root, whether symlinked or recursively copied/uploaded).

Therefore, when building the site, it should be possible to specify an alternate base for media files. For now we can just assume an external base follows the same directory structure as the typical `media/` directory, i.e. no custom mapping function is needed (but in the future this could be implemented, if a desired CDN has its own URL schema).

If provided, we'll require(*) that the provided URL base is absolute, ex. `https://media.hsmusic.wiki/`. It may also specify a directory to act as the root path, ex. `https://hsmusic.wiki/media/`.

<sub>(*) Note: this is to make programming more simple—it's not necessary to code something which is relative or doesn't include the URL unless we specifically want to support uploading one exact build to multiple locations, which isn't a goal I'm tying this feature with right now. "Deploy to multiple locations" is kind of non-specific and less interesting to me than "create a fully-offline, single-download build", which is already covered by just copying the media folder into the output directory (within a zip archive).</sub>

(This is "dev friendliness" because - personally for me - it makes running a distinct preview site, with its own locale builds like `es` and not fudging "`preview-en`", easier lol.)

**Comment by towerofnix** [comment]
Possibly related to #11 but I need to figure out what the frick that issue was even talking about.

**Comment by towerofnix** [comment]
This'll be nice to have after #196 — running thumbnail generation an altogether different (local) server could prove useful for testing and development.

**Comment by towerofnix** [comment]
https://preview.hsmusic.wiki
